### ISSUE TITLE
After 0.3.18, can now toggle Boost mode but not read current state

### ISSUE THREAD

**Issue by flyoffacliff** [georgezhao2010/midea_ac_lan#265]
#142 is not fully fixed. I tested it on two types of Midea units, and It's true that selecting “Boost” from the dropdown now enables boost mode on the units now, when it did not work before (nice job!), but after a few seconds the dropdown selection in Home Assistant switches back to “None” (boost mode stays on and the official app reflects this). Boost mode can be turned off from HA by selecting one of the modes besides “None” and “Boost” that my unit does not support, such as “Eco” or “Comfort”. Selecting “None” does nothing because HA thinks that’s the current mode. (Many systems do not have all these modes so I'm not sure why they are always in the drop down menu, but that's a different issue).

So basically: Your integration can now turn boost off/on, but not read the current state.

**Comment by georgezhao2010** [comment]
Well, turn on debug mode of thsi integration , then turn the Boost mode on and off anywhere, make sure the log file records the changes before and after Boost on, and put the log here or email it to me.

**Comment by georgezhao2010** [comment]
You could clone this integration and replace custom_componnents/midea_ac_lan into your HA and see if it can read the boost mode.

**Comment by georgezhao2010** [comment]
It may be fixed in v0.3.19?
